docs/sources/flow/reference/components/otelcol.exporter.otlphttp.md

```diff
@@ -177,4 +179,4 @@ Connecting some components may not be sensible or components may require further
 Refer to the linked documentation for more details.
 {{< /admonition >}}
 
-<!-- END GENERATED COMPATIBLE COMPONENTS -->
```

Comment:
I am very confused why this is flagged has a change

Reply:
It's tagged in the previous file as well. No idea. I don't see a difference.... maybe an empty line was removed? Git does odd things when empty lines are removed.